docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearning.cs

```diff
@@ -1,123 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.IO;
-using System.Linq;
-using Microsoft.ML;
-using Microsoft.ML.Data;
-using Microsoft.ML.Transforms;
-
-namespace Samples.Dynamic
-{
-    public static class ResnetV2101TransferLearning
-    {
-        /// <summary>
-        /// Example use of Image classification API in a ML.NET pipeline.
-        /// </summary>
-        public static void Example()
-        {
-            var mlContext = new MLContext(seed: 1);
-
-            var imagesDataFile = Path.GetDirectoryName(
-                Microsoft.ML.SamplesUtils.DatasetUtils.DownloadImages());
-
-            var data = mlContext.Data.LoadFromEnumerable(
-                ImageNetData.LoadImagesFromDirectory(imagesDataFile, 4));
-
-            data = mlContext.Data.ShuffleRows(data, 5);
-            var pipeline = mlContext.Transforms.Conversion.MapValueToKey("Label")
-                .Append(mlContext.Transforms.LoadImages("ImageObject", null,
-                    "ImagePath"))
-                .Append(mlContext.Transforms.ResizeImages("Image",
-                    inputColumnName: "ImageObject", imageWidth: 299,
-                    imageHeight: 299))
-                .Append(mlContext.Transforms.ExtractPixels("Image",
-                    interleavePixelColors: true))
-                .Append(mlContext.Model.ImageClassification("Image",
-                    "Label", arch: DnnEstimator.Architecture.ResnetV2101, epoch: 4,
```

Comment:
Maybe add a comment around here saying that just by changing/selecting InceptionV3 here instead of ResnetV2101 you can try a different architecture/pre-trained model. :)